### ISSUE TITLE
Deploy as an opam package

### ISSUE THREAD

**Issue by mattiasdrp** [coq/bot#201]
Would it make sense to deploy at least bot-components as an opam package?

**Comment by Zimmi48** [comment]
Absolutely! That was the plan initially (for the bot-components part). But the API may still evolve a lot, although if it gets users, then we can take their needs into account to make it evolve and have discussions on what it should look like.

Before publishing it on opam, I should make a new release though (the last one was a long time ago).

Are you interested in building a bot based on this library?

**Comment by Zimmi48** [comment]
Yes, I'm interested! OK great if the vendoring method works for you to start with. This should give us more leeway to try to improve the API based on your experience before publishing on opam. Also, there might be some things in the coqbot part that should in fact be reusable and moved to the bot-components part.
